src/libs/numberFormat/index.native.js

```diff
@@ -0,0 +1,14 @@
+// we only need polyfills for Mobile.
+import '@formatjs/intl-getcanonicallocales/polyfill';
+import '@formatjs/intl-locale/polyfill';
+import '@formatjs/intl-pluralrules/polyfill';
+import '@formatjs/intl-numberformat/polyfill';
+
+// Load en Locale data
+import '@formatjs/intl-numberformat/locale-data/en';
+
+function numberFormat(locale, number, options) {
```

Comment:
Also, is there a way to not have to define this twice? And if there is, do we think is worth it?

Reply:
We just need to import those files for mobile(native) platforms. I think it can be done via webpack. But for locale https://github.com/Expensify/Expensify.cash/pull/2208#pullrequestreview-629374884

Reply:
I understand but the `numberFormat` is defined duplicate in 2 places.
(also, that link is supposed to send me where exactly? It takes me here again)

Reply:
Yeah. So I am saying that we can load the imports via `webpack` and remove two files. 

And for locale probably
> A proper way would be when the user switches a locale, load the file remotely.

Reply:
> Yeah. So I am saying that we can load the imports via webpack and remove two files.

I am not sure I know how this works, can you explain?

Reply:
Sorry, I thought I could just load the files via `webpack` config but I don't think either if there is a way to do that.
In that case, we can just put the `numberFormat` function in one js file and use that in both index.js and index.native.js. 
let me know if there is something better.

Reply:
You can import the `src/libs/numberFormat/index.js ` which already defines the function and use it here, no?

Reply:
`index.js` will be replaced with `index.native.js` so we need a new file. then we import from that file. Or better would be to create a file called `lib/polyfills/index.native.js` and import the packages there. this will allow adding all the polyfills at a single location and we import this file in app.js.

Reply:
> index.js will be replaced with index.native.js

Huh? Why? 

What I am saying is:
- The method is in index.js
- index.native.js would import it and use it

Reply:
Ok. let me test it first to be sure. My assumption is that we replace index.js files with index.native.js on mobile platforms.

Reply:
@iwiznia. I tested it. My assumptions are correct. index files are replaced with platform-specific index files(such as index.android.js). So I can make a common base.js file. or let's just leave it like this as this is just two lines of code. We had to create two files. 

1. Index.js
2. index.android.js
To keep it dry, we can create one more base.js, But I think that will be totally overuse of DRY for just two lines. Anyways each platform only gets the file in the bundle.